rpcs3/CMakeLists.txt

```diff
@@ -152,6 +152,11 @@ elseif(UNIX)
             COMMAND ${CMAKE_COMMAND} -E copy_directory
             ${CMAKE_SOURCE_DIR}/bin/git $<TARGET_FILE_DIR:rpcs3>/git)
 elseif(WIN32)
+    if(MSVC)
+        add_custom_command(TARGET rpcs3 POST_BUILD
+            COMMAND "${CMAKE_COMMAND}" -E copy "${PROJECT_BINARY_DIR}/3rdparty/OpenAL/openal-soft/$<CONFIG>/OpenAL32.dll" "$<TARGET_FILE_DIR:rpcs3>"
+            COMMAND "${CMAKE_COMMAND}" -E remove "${CMAKE_SOURCE_DIR}/bin/rpcs3.exe")
```

Comment:
Why do you have to remove this?

Reply:
It is not needed to be remove it but only to exclude it in the copy made by the next command:
`COMMAND "${CMAKE_COMMAND}" -E copy_directory "${CMAKE_SOURCE_DIR}/bin" "$<TARGET_FILE_DIR:rpcs3>"`
But that command has no option to exclude files (in our case only `rpcs3.exe`) so decided to use a dirty solution removing it before the copy.
I could create an external cmake script to replace the `copy_directory` and make a copy excluding the rpcs3.exe file but is seems to me a complicated workaround. An alternative is also to rename the rpcs3.exe before the copy and then restore it after the copy. But even this solution seems to me horrible

Reply:
Why should it exclude the exe?

Reply:
because `rpcs3.exe` is built under `rpcs3\build-msvc\bin` and the post build action, copying the `rpcs3\bin` in `rpcs3\build-msvc\bin`, will override the just compiled `rpcs3.exe` with the one in `rpcs3\bin`.
I'm looking for a solution more similar to the one used for UNIX where the override is avoided.

Reply:
Made a cleanup on post build actions for Windows on cmake solution. As for Unix and Apple, only needed resource files are copied from `rpcs3/bin` folder